### DISCUSSION TITLE
Possible to get schematic for Analog IO

### DISCUSSION THREAD

**Discussion by Jaxc** [DatanoiseTV/PicoDSP-Hardware#6] [Q&A]
Hello

Would it be possible to get the schematic for analog IO(the 8 adc inputs and the stereo output)? I'd like to design something around it but without it it is hard to figure out how to interface it. :) 

Thanks in advance

**Comment by DatanoiseTV** [comment]
Hi,
The ADC inputs have an RC filter on the input (100 Ohm, 33nF).
The Outputs have an RC filter on the output (470 Ohm, 2.2nF - same as in the application example of the PCM5102A).

The Pinout is
<img width="173" alt="Screenshot 2023-06-01 at 08 56 18" src="https://github.com/DatanoiseTV/PicoADK-Hardware/assets/6614616/6d7acd18-7d78-4964-9129-d607351a8a39">


Hope that helps.

Cheers
